services/src/main/java/org/keycloak/authorization/protection/permission/PermissionTicketService.java

```diff
@@ -81,23 +81,23 @@ public Response create(PermissionTicketRepresentation representation) {
             throw new ErrorResponseException("invalid_permission", "created permissions should have scope or scopeName", Response.Status.BAD_REQUEST);
         if (representation.getRequester() == null && representation.getRequesterName() == null)
             throw new ErrorResponseException("invalid_permission", "created permissions should have requester or requesterName", Response.Status.BAD_REQUEST);
-         
```

Comment:
I am sorry for the style changes. I can't seem to change this because git doesn't see any file changes.